libraries/AC_WPNav/AC_Loiter.cpp

```diff
@@ -16,8 +16,9 @@ extern const AP_HAL::HAL& hal;
 const AP_Param::GroupInfo AC_Loiter::var_info[] = {
 
     // @Param: ANG_MAX
-    // @DisplayName: Loiter Angle Max
-    // @Description: Loiter maximum lean angle. Set to zero for 2/3 of PSC_ANGLE_MAX or ANGLE_MAX
+    // @DisplayName: Loiter pilot angle max
+    // @Description{Copter, Sub}: Loiter maximum pilot requested angle lean angle. Set to zero for 2/3 of PSC_ANGLE_MAX/ANGLE_MAX. The maximm vehicle lean angle is still set by PSC_ANGLE_MAX/ANGLE_MAX
```

Comment:
I guess "angle lean angle" should just be "lean angle"

Reply:
fixed, this one was even worse than my very low bar, sorry.